src/ImageSharp/Formats/Jpg/JpegDecoderCore.cs

```diff
@@ -559,42 +547,42 @@ private void ProcessDefineHuffmanTablesMarker(int remaining)
             }
         }
 
-        private void ProcessDefineHuffmanTablesMarkerLoop(ref Huffman huffman, ref int remaining)
+        private void ProcessDefineHuffmanTablesMarkerLoop(ref HuffmanTree huffmanTree, ref int remaining)
```

Comment:
But:
`ProcessDefineHuffmanTablesMarkerLoop()` really should be moved to `HuffmannTree`. I'm already working on a decoder code cleanup to make further optimizations possible. I will take care of this too. Thanks for noticing the anomaly! :)